native/cocos/renderer/pipeline/custom/NativeExecutor.cpp

```diff
@@ -1136,23 +1140,46 @@ struct RenderGraphUploadVisitor : boost::dfs_visitor<> {
             PmrFlatMap<NameLocalID, ResourceGraph::vertex_descriptor> resourceIndex(ctx.scratch);
 
             resourceIndex.reserve(subpass.rasterViews.size() * 2);
-            for (const auto& [resName, rasterView] : subpass.rasterViews) {
+            for (const auto& [resourceName, rasterView] : subpass.rasterViews) {
+                auto resName = resourceName;
                 const auto resID = vertex(resName, ctx.resourceGraph);
                 auto ragId = ctx.fgd.resourceAccessGraph.passIndex.at(vertID);
                 const auto& attachments = ctx.fgd.resourceAccessGraph.access[ragId].attachmentStatus;
-                auto resIter = std::find_if(attachments.begin(), attachments.end(), [resID](const AccessStatus& status) {
-                    return status.vertID == resID;
-                });
-
                 auto slotName = rasterView.slotName;
                 if (rasterView.accessType == AccessType::READ || rasterView.accessType == AccessType::READ_WRITE) {
                     slotName.insert(0, "__in");
```

Comment:
I think `__in` prefix is not needed?

Reply:
In shdc we process input attachment eg `c0` in `#pragma subpassColor c0` into `__inc0`.
Same time user coding like: `pass.addRasterView(user_defined_c0_name, "c0", ...)` according to effect, so it's still `c0` in rasterView.slotName, which does not match what it is in layoutGraph.attributeIndex.
Not sure if name processing is neccesarry.

Reply:
I think in c++ and effect,  __in is not used. `__inc0` is an internal variable and will only be used by `subpassLoad`.

Reply:
In that case, I think the name is not needed?